README.md

```diff
@@ -1,3 +1,45 @@
-# rhpam-7-openshift-image
-Red Hat Process Automation Manager 7 OpenShift images
+# Red Hat Process Automation Manager 7 OpenShift images
 
+This repository contain all the image descriptors and files necessary to build the RHPAM images.
+Also includes the application templates.
+
+
+### Repo structure:
+
+**rhpam-7-openshift-image** \
+├── **[businesscentral](businesscentral)**: Business Central image descriptor files.\
+├── **[businesscentral-indexing](businesscentral-indexing)**: Business Central Indexing (elasticsearch) image descriptor files.\
+├── **[businesscentral-monitoring](businesscentral-monitoring)**: Business Central Monitoring image descriptor files. \
+├── **[controller](controller)**: RHPAM Controller  image descriptor files. \
+├── **[kieserver](kieserver)**: Execution Server image descriptor files \
+├── **[quickstarts](quickstarts)**: quickstarts used as example in some of the application templates. \
+├── **[smartrouter](smartrouter)**: RHPAM Smart Router image descriptor files. \
+└── **[templates](templates)**: Application templates, used to instantiate a new RHPAM environment on OpenShift. \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;├── [contrib](templates/contrib) \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;&nbsp;└── [jdbc](templates/contrib/jdbc): Examples about how to add third party jdbc driver on RHPAM images. \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── [docs](templates/docs):  Application templates docs generated by the [gen_template_docs.py](https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/gen-template-doc/gen_template_docs.py) script. \
+
+Inside each image directory you will find the following files:
+
+ - **container.yaml**: used by OSBS builds.
+ - **content_sets.yaml**: define the yum repositories needed to install dependencies for the image.
+ - **branch-overrides.yaml**: overrides file which uses the latest stable version for external dependencies. Module.yaml and image.yaml always refer to master branch.
```

Comment:
I know it's the beginning of a sentence, so normally capitalized, however the file module.yaml starts with a lower-case 'm'. So perhaps change the sentence to: "The module.yaml and image.yaml files always refer to the master branch."